docs/UserGuide.adoc

```diff
@@ -610,16 +612,54 @@ The Order Panel will also be updated and you can see from the image below that t
 image::Serve_After.png[]
 |===
 
+// tag::bill[]
 [[Bill]]
-==== Calculating the bill of an order : `bill`
+==== Calling the bill of an order : `bill`
 
-Calculates the bill of an order from the current selected table and displays the receipt. +
-Format: `bill`
+Now that all the orders have been served and the customer is ready to pay for the meal, you can call the
+bill to obtain the customer's receipt.
 
-Examples:
+****
+*Format*: `bill` +
+*Shortcut*: `b`
+****
+
+[IMPORTANT]
+This command is irreversible. Once the command is executed, all orders and the table occupancy will be cleared to allow
+new customers to order at the table. The money earned from the bill will automatically be added in the
```

Comment:
new customers to *use* the table. The money earned from the bill will automatically be added *to*